cmd/root.go

```diff
@@ -147,18 +157,16 @@ func rootCmd(o *options.Options) error {
 		return repoResult.Checks[i].Name < repoResult.Checks[j].Name
 	})
 
-	if o.Format == options.FormatDefault {
-		for checkName := range enabledChecks {
-			fmt.Fprintf(os.Stderr, "Finished [%s]\n", checkName)
-		}
-		fmt.Println("\nRESULTS\n-------")
+	for checkName := range enabledChecks {
+		fmt.Fprintf(os.Stdout, "Finished [%s]\n", checkName)
 	}
 
 	resultsErr := pkg.FormatResults(
 		o,
 		&repoResult,
 		checkDocs,
 		pol,
+		output,
```

Comment:
can we avoid changing the signature of this function? this is one of our more public functions.